docs/guide/usage/mqtt_topics_and_messages.md

```diff
@@ -82,6 +82,48 @@ Publishing messages depends on the MQTT client you use. For example to publish a
  mosquitto_pub -t 'zigbee2mqtt/0x0fffffffffffffff/set' -m '{ "state": "ON" }'
 ```
 
+### directly reading/writing attributes
```

Comment:
```suggestion
#### Directly reading/writing ZCL attributes
```

Reply:
So I'm not entirely sure about the terminology but isn't the ZCL only the standard attributes that every device has? (as in, this is "the" spec).

Are any other attributes that manufacturers added themselves also still "ZCL attributes"?